tests/item-counter.test.js

```diff
@@ -0,0 +1,30 @@
+// import getShows from '../src/modules/shows-api-helpers';
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
Very good point 👍